schemes/check/CMakeLists.txt

```diff
@@ -51,15 +51,18 @@ set(SOURCES
     check_noop.f90
 )
 
-#------------------------------------------------------------------------------
-# Add the auto-generated caps
-add_custom_command(
-    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/check_test_cap.f90
-    DEPENDS ${CCPP_MKCAP}
-    COMMAND ${CCPP_MKCAP} -o ${CMAKE_CURRENT_BINARY_DIR}/check_test_cap.f90
-                             ${CMAKE_CURRENT_SOURCE_DIR}/scheme.xml
-)
-list(APPEND SOURCES ${CMAKE_CURRENT_BINARY_DIR}/check_test_cap.f90)
+# DH the mechanism to create caps from XML is no longer supported; todo:
+# replace with a test that parses a simple metadata table in check_test.f90
+# and creates check_test_cap.f90 - also needs a variable definition file.
+##------------------------------------------------------------------------------
```

Comment:
I guess that depends on what the developers agreed on. Since we decided to use issues in the future, I agree that we (I) need to update our methods in the future. We need to be able to reference the part of the code in the issue, though (github probably lets you do this, but I don't know yet how).